stdlib/src/builtin/int_literal.mojo

```diff
@@ -269,6 +269,47 @@ struct IntLiteral(
         """
         return self
 
+    @always_inline("nodebug")
+    fn __divmod__(self, rhs: Self) -> Tuple[Self, Self]:
```

Comment:
non-blocking: maybe in a followup: Maybe it's time for a `DivModable` trait or something. Then we could hook these up into the builtin `divmod` function.